src/index.ts

```diff
@@ -47,6 +47,11 @@ export interface Options {
    * File path to use in error messages, React display names, and source maps.
    */
   filePath?: string;
+  /**
+   * If specified, we generate a few less things (for example the jsx
```

Comment:
Suggestions wanted.

Reply:
How about this: "If specified, omit any development-specific code in the output."

It can be changed later if more use cases come up that don't exactly fit that.